### PR TITLE
Update links in README

### PR DESCRIPTION
**Pull request by dolph**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.com/IBM/MAX-Image-Completer.svg?branch=master)](https://travis-ci.com/IBM/MAX-Image-Completer)
+[![Build Status](https://api.travis-ci.com/IBM/MAX-Image-Completer.svg?branch=master)](https://travis-ci.com/IBM/MAX-Image-Completer)
```
Comment (ptitzler) [inline README.md]: What's the motivation/reasoning for this change?

https://travis-ci.com/IBM/MAX-Image-Completer uses 

image

Comment (bdwyer2) [reply README.md]: https://travis-ci.com/IBM/MAX-Image-Completer.svg?branch=master redirects to https://api.travis-ci.com/IBM/MAX-Image-Completer.svg?branch=master

Comment (ptitzler) [reply README.md]: Yes, but if they wanted users to point directly to the `api...` URL, wouldn't they have used it in the first place in the displayed URL? We don't have any insights into their content delivery strategy ...

Comment (dolph) [reply README.md]: Well, it's not just a redirect, it's actually a permanent redirect (HTTP 301). I agree they're currently self-contradictory, though, and happy to revert this particular change until their docs change.

I made this change to all the AIDA repos, so I'd change it back in all of them, as well. This particular repo just didn't have any of the other similar updates that I made to links.

Comment (stevemart) [reply README.md]: @ptitzler yeah, it's a permanent redirect, not sure why their UI hasn't been updated yet.

FWIW, @dolph created similar PRs for 101 different repos for code patterns so if there's something you want changed or reverted just let us know.

Comment (ptitzler) [reply README.md]: Well, there seems little benefit in pushing these changes (there's no obvious indication that they will "deprecate" the old build status endpoint and any  potential performance gain (due to the avoided redirect) should be negligible) that outweighs the potential risk. I've checked the Travis CI API doc but didn't find any relevant documentation for the the api... endpoint we are getting redirected to.

Comment (dolph) [reply README.md]: I'm going to close this PR and revert this change in other repos, likely with other changes (I don't think it's worth a bunch of new PR's to revert this immediately).

**Pull request closed without merging**